packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1030,6 +1039,32 @@ class RawScrollbar extends StatefulWidget {
   /// If null, defaults to Color(0x66BCBCBC).
   final Color? thumbColor;
 
+  /// The preferred smallest size the scrollbar can shrink to when the total
+  /// scrollable extent is large, the current visible viewport is small, and the
+  /// viewport is not overscrolled.
+  ///
+  /// The size of the scrollbar may shrink to a smaller size than [minThumbLength] to
+  /// fit in the available paint area. E.g., when [minThumbLength] is
+  /// `double.infinity`, it will not be respected if
+  /// [ScrollMetrics.viewportDimension] and [mainAxisMargin] are finite.
+  ///
+  /// Mustn't be null and the value has to be greater or equal to
+  /// [minOverscrollLength], which in turn is >= 0. Defaults to 18.0.
+  final double minThumbLength;
+
+  /// The preferred smallest size the scrollbar can shrink to when viewport is
+  /// overscrolled.
+  ///
+  /// When overscrolling, the size of the scrollbar may shrink to a smaller size
```

Comment:
We should mention that overscrolling is made possible with BouncingScrollPhysics here as well.

Reply:
Just out of curiosity, are there any other `ScrollPhysics`s that allow for overscrolling?

Reply:
I think that is the only one currently. :)

Reply:
I see. I've made the changes btw, would you like to modify anything else?